src/main/java/codurance/academyfinalboy/backend/model/team/TeamService.java

```diff
@@ -21,7 +21,7 @@ public Long createTeam(Long userId, String name, String description) {
     return teamRepository.save(team);
   }
 
-  public TeamWithMembers getTeam(Long teamId) throws Exception {
+  public TeamView getTeam(Long teamId) throws Exception {
```

Comment:
I think when you merged from main it changed this for you but the current version is "TeamWithMembers"

Reply:
So this is a discussion we had in which we kinda liked the name TeamView again because now it also has te activities inside, but is discussable

Reply:
Oh you are right, I did not realise  about that 👍